routes/notices.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const asyncMiddleware = require('../middleware/asyncMiddleware')
+const express = require('express')
+const router = express.Router()
+const EntityCoordinates = require('../lib/entityCoordinates')
+const validator = require('../schemas/validator')
+
+router.post(
+  '/',
+  asyncMiddleware(async (request, response) => {
```

Comment:
suggest using the new style of having the handler as a separate method that is just referenced from the router call.  Looks better, easier to read and more testable.